siliconcompiler/core.py

```diff
@@ -2791,7 +2791,11 @@ def _generate_summary_image(self, input_path, output_path):
         # "final" metrics regardless of flow would be handy
         totalarea = self.get('metric', 'totalarea', step='export', index='1')
         if totalarea:
-            metrics['Area'] = f'{totalarea:.2f} um^2'
+            # SI-ify
+            if totalarea < 1e3:
+                metrics['Area'] = f'{totalarea:.2f} um^2'
+            else:
+                metrics['Area'] = f'{totalarea / 1e3:.2f} mm^2'
```

Comment:
this is an area, so it should be 1e6 to scale to mm^2

Reply:
oh yeah, math

that explains why GCD was coming back so large